src/logger.js

```diff
@@ -191,22 +190,26 @@ function logToConsole(type, sender, msgShort, _extras) {
     if (logger.graylogger === null) logger.initGraylog()
 
     // smallExtras added because just sending the extras json can lead to trouble with indexing on the graylog server if it's not consistent
-    let smallExtras = {
-      sender: sender,
-      type: type,
-    }
+    const smallExtras = { sender, type }
     if (extras.user_id) smallExtras.user_id = Number(extras.user_id) || 0
     if (extras.userId && !smallExtras.user_id) smallExtras.user_id = Number(extras.userId) || 0
     if (extras.filename) smallExtras.filename = extras.filename
 
     // limit size of msgLong to avoid excessive storage consumtion and failures
     // (up to 32766 byte strings should be possible, but 10k characters is already plenty for reasonable logging output)
-    const msgLongGray = (typeof msgLong === 'string' ? msgLong.substring(0,10000) : null)
+    const msgLongGray = (typeof msgLong === 'string' ? msgLong.substring(0, 10000) : null)
 
     try {
-      logger.graylogger.log(msgShort, msgLongGray, smallExtras)
+      const leveledLogFunction = logger.graylogger[type.toLowerCase()] || logger.graylogger.info
```

Comment:
this file is the most strange implementation of logger I've ever seen

Reply:
😅